must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -23,8 +23,8 @@ noobaa_cli+=(noobaa obc list)
 # Run the Collection of Noobaa cli using must-gather
 for cli in "${noobaa_cli[@]}"; do
     echo "collecting dump of ${cli}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
-    COMMAND_OUTPUT_FILE="${NOOBAA_COLLLECTION_PATH}"/raw_output/"${cli}"// /_}
-    { timeout 120 "${cli}" >> "${COMMAND_OUTPUT_FILE}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+    COMMAND_OUTPUT_FILE="${NOOBAA_COLLLECTION_PATH}"/raw_output/${cli// /_}
```

Comment:
Interesting that this sneaked in a fix in this line too... This should have been a different commit. and possibly different PR. But since this was done in master like this and this is only a backport, let's take it ...